kubeflow/fairing-py/tf_smoke.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+
+# Copyright 2018 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# kubeflow/training
+
```

Comment:
What is this being used for?

Reply:
Same as the dockerfile above, just an e2e example. Hoping to have this eventually trained through TFJob